test/utility.cc

```diff
@@ -26,6 +26,9 @@ std::vector<std::string> getRuntimes() {
 #endif
 #if defined(WASM_WASMTIME)
     "wasmtime",
+#endif
+#if defined(WASM_WAMR)
```

Comment:
where `WASM_WAMR` is defined in build targets..? I don't see any test is running correctly for WAMR so far..

Reply:
> where `WASM_WAMR` is defined in build targets..? I don't see any test is running correctly for WAMR so far..

In Proxy-wasm/Build, I locally set the target as WAMR_lib and exculde wasmtime_lib because of the potential symbol conficts. I just run the build test //test/... , where should I define the WAMS_WAMR

Reply:
OK, as long as Envoy test passes, we are fine then for now. 

As in https://github.com/proxy-wasm/proxy-wasm-cpp-host/pull/142#discussion_r613046016, we should fix CI here in another PR to test these runtimes (Wasmtime+WAMR) in GHA.

Reply:
Fine. Thanks a lot.